src/rdkafka_partition.c

```diff
@@ -1942,7 +1942,14 @@ static void rd_kafka_toppar_op (rd_kafka_toppar_t *rktp,
 	rd_kafka_toppar_op0(rktp, rko, replyq);
 }
 
-
+static int rd_kafka_toppar_fwd_app (rd_kafka_toppar_t *rktp)
+{
```

Comment:
* open brace on same line as function.
* maybe indicate that this function doesnt actually forward or change anything, just checks it: `rd_kafka_toppar_is_fwd_app()`
* add doxygen-style docstring:
```
/**
 * @brief